editor/editor_settings.cpp

```diff
@@ -959,27 +959,18 @@ void EditorSettings::create() {
 
 		_create_script_templates(dir->get_current_dir().plus_file("script_templates"));
 
-		if (dir->change_dir("projects") != OK) {
-			dir->make_dir("projects");
-		} else {
-			dir->change_dir("..");
-		}
-
-		// Validate/create project-specific config dir
-
-		dir->change_dir("projects");
-		String project_config_dir = ProjectSettings::get_singleton()->get_resource_path();
-		if (project_config_dir.ends_with("/")) {
-			project_config_dir = config_path.substr(0, project_config_dir.size() - 1);
-		}
-		project_config_dir = project_config_dir.get_file() + "-" + project_config_dir.md5_text();
-
-		if (dir->change_dir(project_config_dir) != OK) {
-			dir->make_dir(project_config_dir);
-		} else {
-			dir->change_dir("..");
+		{
+			// Validate/create project-specific editor settings dir.
+			DirAccess *da = DirAccess::open("res://");
```

Comment:
You can use `DirAccessRef da`, so you don't need to `memdelete()` it (done automatically when it goes out of scope).

Reply:
Also, I *think* we tend to use `DirAccess::create(DirAccess::ACCESS_RESOURCES);` which should be equivalent (didn't check though).